ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -31,13 +39,13 @@ async function buildWebDriverAgent (xcodeVersion) {
   await mkdirp('bundles');
```

Comment:
is this dir created in the root folder or inside cwd?

Reply:
I changed it to be fully qualified.